packages/nimble-components/src/table/components/header/template.ts

```diff
@@ -14,15 +18,27 @@ export const template = html<TableHeader>`
         @mousedown="${(_x, c) => !((c.event as MouseEvent).detail > 1)}"
     >
         <slot></slot>
-        ${'' /* Omit title attribute for sort indicators because aria-sort is set on the 1st sorted column */}
-        ${when(x => x.sortDirection === TableColumnSortDirection.ascending, html`
-            <${iconArrowUpTag} class="sort-indicator" aria-hidden="true"></${iconArrowUpTag}>
+        ${'' /* Set aria-hidden="true" on sort indicators because aria-sort is set on the 1st sorted column */}
```

Comment:
This comment was a bit confusing to me. I'm not sure why we would omit the title, which could be helpful for sighted people, because we don't want the icon to be present in the accessibility tree. Since the sort icons already have `aria-hidden="true"` set on them, I didn't see any harm in applying a title. Let me know if I'm overlooking something.

Reply:
[Here's the related discussion](https://github.com/ni/nimble/pull/1328#discussion_r1248069793) from my original label provider PR.

So the original thought was that setting title on the sort icons was redundant due to aria-sort. However I don't think I considered the fact that they were already set as `aria-hidden` so that didn't really apply.
I think your logic (could be useful for sighted people) makes sense so I'm fine adding title.